moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -65,3 +76,370 @@ export const resolveHasSubprojects = (array) => {
   }
   return "No";
 };
+
+/**
+ * Various custom components for Material Table
+ */
+export const useTableComponents = ({
```

Comment:
these new hooks are a bit 😵 but i'm not sure how to make them better 🤷

Reply:
I think this is fine and works within the boundaries of what it takes to optimize this table! It also maintains the different pieces that fit into the material table component which I think is still straightforward to see.

Reply:
ok—thanks, Mike!